scripts/apps/search/directives/ItemList.js

```diff
@@ -718,11 +718,11 @@ export function ItemList(
                                 ));
 
                     var elem = React.createElement('div',
-                        {
-                            className: 'dropdown__menu more-activity-menu',
-                            onMouseOver: this.toggle,
-                            onMouseLeave: this.close
-                        },
+                                                   {
```

Comment:
But, it fails for the exact opposite reason now. So, I'm going to remove it :)
![ss_today](https://cloud.githubusercontent.com/assets/13354440/21127232/9304d54e-c145-11e6-8d37-ac4d32ebc3ea.jpg)